atlp-rwanda/winners-c8-bn-be
# chore-node-scope-eslint-configuration

## What does this PR do? ## 
1. it set up the configuration of eslint in this project to enforce consistency and avoid mistakes!!!!
2.  it set script for running this configuration

## How should this be manually tested? ##
1. Clone this repository
2. run `npm install` to bring its modules and packages
3. run `npm run start-lint` to lint the project and check if there any rules violated!
4. then it will show you what wrong with your codes! based on what you have to modify?
